packages/graph/hash_graph/lib/graph/src/store/mod.rs

```diff
@@ -263,9 +263,9 @@ pub trait PropertyTypeStore:
     async fn get_property_type(
         &self,
         query: &Expression,
-    ) -> Result<Vec<PersistedPropertyType>, QueryError> {
-        self.read(query).await
-    }
+        data_type_resolve_depth: u8,
```

Comment:
Is there a reason we're using "query_depth" in the structs but "resolve_depth" here?

Reply:
Also generally is there a reason we're not just taking the structs? I'm a little concerned about us destructuring into a bunch of U8s, accidental parameter re-ordering could be a bad time

Reply:
I changed this to use the `Query*` structs: 66aa16e853ee9c97839ae35a27ece4b642b7dc2b